tests/proxy/mocks/mc/price-benchmarks/price-insights.json

```diff
@@ -0,0 +1,20 @@
+{
+	"results": [
+		{
+			"productView": {
+				"offerId": "gla_103572",
+				"title": "Dr. Seuss The Grinch Stole Christmas Ugly Sweater xl",
+				"currencyCode": "USD",
+				"priceMicros": "34950000",
+				"id": "online:en:US:gla_103572"
+			},
+			"priceInsights": {
+				"suggestedPriceMicros": "135680000",
+				"suggestedPriceCurrencyCode": "USD",
+				"predictedImpressionsChangeFraction": "0.12609300017356873",
+				"predictedClicksChangeFraction": "0.508745014667511",
+				"predictedConversionsChangeFraction": "2.3431060314178467"
```

Comment:
The `effectiveness` is missing here

Reply:
The Query itself didn't return it see https://developers.google.com/shopping-content/guides/reports/price-insights. How we get that?

Reply:
Good observation. I noticed that as well. I'll need to see if we need to format the query differently to get that data to return in the API responses. Let's pick that up as part of #2911.